### ISSUE TITLE
Rolls from DDB are not formatted correctly in Foundry

### ISSUE THREAD

**Issue by dcode** [IamWarHead/ddb-game-log#112]
I'm using DDB Game Log v2.0.0 with Foundry 11 and D&D Game System 3.0.3. When I make rolls from the DDB app (android), the card looks like this:
<img width="225" alt="image" src="https://github.com/IamWarHead/ddb-game-log/assets/171574/21c4e3ad-a1b2-4295-ba65-798fd357bded">

However, when I roll within the Foundry character sheet (the updated D&D sheet), it comes out like this, without the overlapping text:

<img width="223" alt="image" src="https://github.com/IamWarHead/ddb-game-log/assets/171574/ba759f53-533f-4652-b9d5-79809f0c8d7e">

Lastly, just to see if there was a difference, I noticed if I go to dndbeyond.com and roll, it looks slightly different than the first, namely the avatar.

<img width="225" alt="image" src="https://github.com/IamWarHead/ddb-game-log/assets/171574/3986b803-65ca-40cb-91de-17bd4a0e3e74">

---

The worse part for my players, is I use the "Foundry to Discord" plugin, and the broken formatting seemingly has broken the card parsing to forward the format.

<img width="400" alt="image" src="https://github.com/IamWarHead/ddb-game-log/assets/171574/760a246d-98dc-4903-96be-655e314b2fc3">

vs

<img width="195" alt="image" src="https://github.com/IamWarHead/ddb-game-log/assets/171574/7380429c-baf8-4516-98e6-dea0133ba180">

---

Let me know if there is more information I can provide to help fix this.

Thanks!

**Comment by AlexHailstone** [comment]
Thanks for the submission I was just about to post the same bug/issue.

**Comment by Hoosjan** [comment]
@dcode and @AlexHailstone I can't get it to work. The module says its connected but it's not showing the dice rolls from the log in Foundry. Did you run in to this and manage to fix it?

**Comment by dcode** [comment]
@Hoosjan I do successfully get my rolls into Foundry VTT from D&D Beyond. There's a couple "gotchas" that I've figured out.

1. The Gamemaster account that has the cobalt cookie must be logged into Foundry for the rolls to come through since the plugin runs in that browser instance.
2. The players need to make sure they've disabled Beyond20 browser plugin if they are using it, as it blocks this plugin from accessing the rolls on D&D Beyond.
3. Rolls made to D&D Beyond from bots like Avrae don't get captured either.
4. You have to have the right subscription tier to get rolls made with the mobile D&D Beyond app.

**Comment by Hoosjan** [comment]
@dcode awesome! Thanks for the swift response. Do you mean a webbrowser for the gm next to the foundry app on the computer?

**Comment by dcode** [comment]
Yes, the GM has to be logged into a web browser because the cobalt cookie resides in that browser session (as far as I can tell, I am not the author).

**Comment by Hoosjan** [comment]
@dcode That did the trick, thanks a lot!

**Comment by CacciaMorta** [comment]
I have not solved this issue, even after login via a web browser. Moreover, after the last update, I cannot see the rolls and this error message appears: "cannot destructure property 'pack' of 'n' as it is undefined". So in this moment it is not working at all.

**Comment by crlngn** [comment]
> I have not solved this issue, even after login via a web browser. Moreover, after the last update, I cannot see the rolls and this error message appears: "cannot destructure property 'pack' of 'n' as it is undefined". So in this moment it is not working at all.

By the last update do you mean Foundry V12?

**Comment by CacciaMorta** [comment]
Yes, but all is solved after an update of DDB-gamelog yesterday.

**Comment by CacciaMorta** [comment]
I was wrong, the problem was not solved. Sorry for the misleading message above. I made confusion with another module.
Importantly, there are no error messages.
